gravitee-apim-console-webui/src/management/api/api-ng-navigation/api-ng-v4-menu.service.ts

```diff
@@ -195,12 +200,6 @@ export class ApiNgV4MenuService implements ApiMenuService {
         targetRoute: 'DISABLED',
       });
     }
-    if (this.permissionService.hasAnyMatching(['api-log-r'])) {
```

Comment:
@Okhelifi there are other places with this do we need to remove them also?
gravitee-apim-console-webui/src/management/api/apis.module.ts L1334 & L1248

Reply:
I think that it's linked to the V2 APIs here we only work on the V4 APIs so I would not touch it. We need to keep those pages available to the V2 APIs or the customers will lose a feature finally